src/content/en/updates/2018/11/webcomponents-v0-deprecations.md

```diff
@@ -36,64 +39,62 @@ modules in 2017.
 - HTML Imports: [Chromestatus Tracker](https://www.chromestatus.com/feature/4507242028072960)  &#124;
 [Chromium Bug](https://crbug.com/766694)
 
-### Why at once?
+### Why removing them at once?
 
-Since then, we also found that in many real-world use cases, Shadow DOM V0,
-Custom Elements V0, HTML Imports were used together, and instead of getting each
- of them migrated respectively, getting all of them migrated to today’s standards
- at once, or at least use the polyfill would make it less troublesome, easier to
- migrate, rather than fighting with every edge case of combination of polyfill
-  and native implementation.
+Chrome usually removes features one by one, but why at once for these features this time?
+Since launched, we found that they are often used together.
+Especially [webcomponentsjs polyfill](https://github.com/WebComponents/webcomponentsjs)
+ was originally designed to simulate the v0 features on other
+browsers and not do so on Chrome being removed each feature randomly.
+For safer/lesser troublesome migration off from the features, we've desided to
+remove them at one.
 
 ## Migration support
-If you are using this features, please consider migrating to the new API or
-upgrading your Polymer library.
-Use `--disable-blink-features=ShadowDOMV0,CustomElementsV0,HTMLImports` for
- testing if your site works without those APIs.
+If your site and/or libirary is using this features, you should upgrade them.
```

Comment:
libirary -> library.

Can you give the article a proofread for typos. I'll hold off on reviewing more.